### ISSUE TITLE
may you clarify what data you used

### ISSUE THREAD

**Issue by Sandy4321** [jrzaurin/RecoTour#3]
may you clarify what data you used
relating to 
'''

I run this on a c5n.4xlarge instance with 40GB of memory and by the time it finished it had almost consumed it all. Previously, when using a c5.4xlarge (30GB) I got the always nice Memory error message "Killed" in the terminal
'''
this
http://jmcauley.ucsd.edu/data/amazon/
easy to get
if yes then which link
this one?
aggressively deduplicated data (18gb) - no duplicates whatsoever (82.83 million reviews)

or 
https://nijianmo.github.io/amazon/index.html
To request for the complete review data as well as the per-category files, you will need to complete this form.

**Comment by Sandy4321** [comment]
I see this error for small data due to memory leaking
but in any case what amazon data you used? 
http://jmcauley.ucsd.edu/data/amazon/
or
https://nijianmo.github.io/amazon/index.html

**Comment by jrzaurin** [comment]
hey

I see that the link in the notebooks is broken, I will fix it. 

I used these data: 
-- | -- 
Movies and TV | 5-core (1,697,533 reviews)
-- | --

you can get it from here: http://jmcauley.ucsd.edu/data/amazon/

Or simply:
```
wget http://snap.stanford.edu/data/amazon/productGraph/categoryFiles/reviews_Movies_and_TV_5.json.gz
```

**Comment by Sandy4321** [comment]
may be use this idea to simulate data 
https://github.com/emeryyi/EMTboost/blob/master/generator.R
Simulation code for Tweedie Gradient Boosting for Extremely Unbalanced Zero-inflated Data
based on
https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_friedman2.html#sklearn.datasets.make_friedman2
